### PR TITLE
Use the in ubuntu-core python3

### PR DESCRIPTION
**Pull request by sergiusens**
LP: #1582513

Signed-off-by: Sergio Schvezov <sergio.schvezov@ubuntu.com>

### CODE REVIEW

File: snapcraft/plugins/python3.py [hunk 1]
```diff
@@ -69,16 +69,18 @@ def schema(cls):
...
             'python3-dev',
+            'python3-pip',
             'python3-pkg-resources',
             'python3-setuptools',
         ])
 
     def env(self, root):
         return [
-            'PYTHONPATH={}'.format(os.path.join(
+            'PYTHONPATH={0}'.format(os.path.join(
```
Comment (kyrofa) [inline snapcraft/plugins/python3.py]: Is there a reason to use `{0}` instead of `{}` here?

Comment (sergiusens) [inline snapcraft/plugins/python3.py]: left over from when I tried to use `:` with multiple entries, so no :-)

**Comment by sergiusens** [comment]
retest this please

**Comment by sergiusens** [comment]
all green :100:

**Comment by kyrofa** [comment]
> all green :100: 

Can you really celebrate that when your tests look like [this](https://github.com/ubuntu-core/snapcraft/pull/510/files#diff-e047afa9d05eda5d5cf933f0e2dc11dcR22)? :stuck_out_tongue:

**Comment by sergiusens** [comment]
@kyrofa I celebrate the fact that the infrastructure work and there was no regression! :wink:

**Comment by kyrofa** [comment]
Hahaha

**Comment by elopio** [comment]
alright, this seems to simplify the code a lot. Do we need a review from a python/pip expert?

You need to readd the tests for py3, and rebase with master.

**Pull request closed without merging**